lib/resources/tasks/run.js

```diff
@@ -1,22 +1,10 @@
 import gulp from 'gulp';
 import browserSync from 'browser-sync';
 import historyApiFallback from 'connect-history-api-fallback/lib';
+import { CLIOptions } from 'aurelia-cli';
```

Comment:
An lint rule could be added to https://github.com/aurelia/tools/blob/e457d2d753e0abdc1a8f75d5d9e79ccb464f4f7c/.eslintrc.json in order to standardise the spacing.